iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/MergingViewIndex.java

```diff
@@ -68,6 +71,29 @@ void checkAndAdd(final Network other) {
         ValidationUtil.checkUniqueIds(other, this);
         // Local storage for mergeable network
         networks.add(other);
+        // Manage DanglingLines
+        other.getDanglingLineStream().forEach(this::checkNewDanglingLine);
+    }
+
+    void checkNewDanglingLine(final DanglingLine dll2) {
```

Comment:
Should be private?

Reply:
Is used by : MergingNetworkListener & DanglingLineAdderAdapter